docs/checkit.js

```diff
@@ -685,30 +699,51 @@ CheckIt.prototype.deleteCalendar = function() {
     
     // Guard against accidental clicks of the delete button
     
+
     var confirmation = confirm("Are you sure you want to delete your calendar?");
-    if (confirmation) {
-        
-        var checkit = this;
-                
-        return checkit.store.getActive().then(function(currentCalendarId) {
+    if (!confirmation) {
+      return Promise.reject("User cancelled the delete operation.");
+    }
+
+    var checkit = this;
+
+    return checkit.store.getActive()
+        // if rejected promise returned catch gets run
+        .catch(function(err) {
+            console.error("Couldn't get active calendar id: " + err);
+            // we don't return a resolved promise here, so after this, 
+            // we exit the function?
```

Comment:
Even if you *were* returning a rejected promise here, you would exit this function at this point.